packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +151,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+        }
+        .day,
+        .interact .day,
+        .week-day {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+            fill: currentcolor;
+            border-color: ${SystemColors.Canvas}
+        }
+        .interact .day:not(.disabled, .inactive):hover {
+            background: ${SystemColors.ButtonFace};
+            color: ${SystemColors.ButtonText};
+            outline: 1px solid ${SystemColors.Highlight};
```

Comment:
In the latest pattern for the styles, `outline` should only be used for focus treatment. This doesn't appear to be a focused state, so it should probably use `border` instead. Is the border needed for the regular styles or only for HC?

Reply:
switch to border-color